LogicLayer/Notifications/Html/Raw/friend-invitation.html

```diff
@@ -0,0 +1,52 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta name="viewport" content="width=device-width"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <title>Friend invitation</title>
+    <link href="styles.css" media="all" rel="stylesheet" type="text/css"/>
+</head>
+
+<body>
+
+<table class="body-wrap">
+    <tr>
+        <td></td>
+        <td class="container" width="600">
+            <div class="content">
+                <table class="main" width="100%" cellpadding="0" cellspacing="0">
+                    <tr>
+                        <td class="content-wrap">
+                            <table width="100%" cellpadding="0" cellspacing="0">
+                                <tr>
+                                    <td class="content-block">
+                                        You have a new friend invite pending.
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td class="content-block">
+                                        Click here to review your friend invitations
```

Comment:
Done

Reply:
It still says "See Friend Invitation**s**". Which also reveals an inconsistency, since in email confirmation the button text is not capitalized. We're not capitalizing every word in any other scenario so I suggest we don't here either. The button text should be "See friend invitation". Make the capitalization consistent across all htmls.

Reply:
Done

Reply:
It's still capitalized in event-cancellation, event-invitation, password-reset.

Reply:
*Doing his best zirrock impersonation* Done!